### PR TITLE
Add missing translations for raidboss and oopsy + small fixes of incorrect regexes

### PR DESCRIPTION
**Pull request by Akurosia**

### CODE REVIEW

**Comment by JLGarber** [comment]
I guess I might as well ask here, what sort of a script are you running to generate the KO translations? I haven't finished putting support for the KO .csv repository into my translation script yet, which is why I didn't have them for Sohm Al hard. (That, and the dungeon isn't on FFLogs, so I couldn't translate that way.) Or was one of the cactbot utilities updated to generate that without me noticing?

File: ui/raidboss/data/05-shb/alliance/the_copied_factory.js [hunk 1]
```diff
@@ -747,12 +747,12 @@
         'Warehouse A': 'Warenlager A',
         'Warehouse B': 'Warenlager B',
         'Warehouse C': 'Warenlager C',
-        'The wall-mounted right arm begins to move\.\.\.': 'Der wandmontierte rechte Arm ist aktiv!',
-        'The wall-mounted flamethrowers activate\.': 'Die wandmontierten Flammenwerfer sind aktiv!',
-        'The wall-mounted left arm begins to move\.\.\.': 'Der wandmontierte linke Arm ist aktiv!',
-        'You hear frenzied movement from machines beneath\.\.\.': 'Die Maschinenwesen zu deinen Füßen bewegen sich!',
+        'The wall-mounted right arm begins to move\\\\.\\\\.\\\\.': 'Der wandmontierte rechte Arm ist aktiv!',
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/alliance/the_copied_factory.js:750]: Maybe we should just remove the dots on the end of both the original text and the replacement?

File: ui/raidboss/data/05-shb/raid/e1s.js [hunk 2]
```diff
@@ -578,7 +578,7 @@
         'Paradise Regained': '复乐园',
         'Pure Beam': '净土射线',
         'Pure Light': '净土之光',
-        'Spear [oO]f Paradise': '乐园之枪',
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e1s.js:581]: oof

Comment (Akurosia) [reply ui/raidboss/data/05-shb/raid/e1s.js:581]: there is nothing here you have to look at <.<

**Comment by Akurosia** [comment]
@JLGarber i am using a custom script based on many other custom scripts xD

**Comment by JLGarber** [comment]
@Akurosia I thought that was likely the case, but I had to ask just in case it was easily shareable. No worries, I'll get mine finished up Sometime Soon and then I'll see about adding it to `translate_fight`.

**Pull request merged**